pyproject.toml

```diff
@@ -354,9 +357,10 @@ disable = [
 
 [tool.pytest.ini_options]
 # sync minversion with pyproject.toml & install.rst
-minversion = "7.0"
-addopts = "--strict-data-files --strict-markers --strict-config --capture=no --durations=30 --junitxml=test-data.xml"
-empty_parameter_set_mark = "fail_at_collect"
+minversion =  "7.0"
+addopts = "--strict-markers --strict-config --capture=no --durations=30 --junitxml=test-data.xml"
+# Pickle tests will fail when the pickle data files are not found if this is set to fail_at_collect
+empty_parameter_set_mark = "skip" 
```

Comment:
I would prefer to keep this as `fail_at_collect` and adjust not parameterize the once pickle test that needs the legacy files